Userland/Libraries/LibRegex/RegexParser.cpp

```diff
@@ -935,20 +931,20 @@ bool ECMA262Parser::parse_internal(ByteCode& stack, size_t& match_length_minimum
         return parse_pattern(stack, match_length_minimum, true, true);
     }
 
-        ByteCode new_stack;
-        size_t new_match_length = 0;
-        auto res = parse_pattern(new_stack, new_match_length, false, false);
-        if (m_parser_state.named_capture_groups_count > 0) {
-            reset();
-            return parse_pattern(stack, match_length_minimum, false, true);
-        }
+    ByteCode new_stack;
```

Comment:
What's up with all the indentation change?

Reply:
It was in an else block before, and is now in the main function block instead, decreasing the indentation by one

Reply:
And the diff isn't showing it the best way as it seems

Reply:
The indentation changes should be squashed in to the commit that removed the `else` blocks (`LibRegex: Remove some else-after-returns`) instead of this one.

Reply:
Oh did that move to the wrong one?
I will fix that up right away!
(Damn VSCode not showing white-space changes)